xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -243,6 +249,7 @@ private void fillObjectSummary(ObjectSummary objectSummary, Document doc, BaseOb
                 objectSummary.setHeadline(serializePropertyValue(xwikiObject.get(propertyNames[0])));
```

Comment:
here is the place where I miss to handle emails as I do not get at the type here. this is only a problem when the email is the first property of a class, but nevertheless ...